src/bci_build/package/python.py

```diff
@@ -74,8 +74,10 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
         "env": {
             "PYTHON_VERSION": py3_ver_replacement,
             "PIP_VERSION": pip3_replacement,
-        }
-        | ({"PATH": "$PATH:/root/.local/bin"} if has_pipx else {}),
+            "PIPX_HOME": "/opt/pipx",
```

Comment:
this should only be set if it is a container with pipx. also `/opt/pipx` is nonstandard. please use either `/var/lib/pipx` or `/usr/local/lib/pipx` instead (I think the 2nd is preferred).

Reply:
I took the path from pipx docs, but indeed, `/usr/local/lib/pipx` feels more correct.